src/controllers/OauthController.js

```diff
@@ -0,0 +1,33 @@
+import { z } from 'zod';
+
+import OauthService from '../services/OauthService.js';
+
+import UserService from '../services/UserService.js';
+
+async function login(_request, response) {
```

Comment:
O `_` é usado para indicar que o parâmetro foi ignorado, mas ele está sendo usado no corpo da função.

```suggestion
async function login(request, response) {
```